pom.xml

```diff
@@ -1181,7 +1204,7 @@ under the License.
 			<plugin>
 				<groupId>org.apache.maven.plugins</groupId>
 				<artifactId>maven-enforcer-plugin</artifactId>
-				<version>1.4.1</version><!--$NO-MVN-MAN-VER$-->
+				<version>${enforcer.version}</version><!--$NO-MVN-MAN-VER$-->
```

Comment:
How about defining the plugin in the plugin management section, rather than using a variable? I think that is the more maintainable solution in the log run.